### DISCUSSION TITLE
Why Can't we download old versions of the repo as a complete .zip of any version of the Repo in GitHub? it only works for the latest version only.

### DISCUSSION THREAD

**Discussion by Husam-AbuZina** [community/community#130727] [Repositories] [answered]
### Select Topic Area

Question

### Body

I am a fellow software developer, who mainly works with web development. I wonder why I can't download a .zip file of any version of my GitHub Repo, Instead of only getting the opportunity to download the latest version of the repo. And instead of downloading the latest version and just copying the old code for me file by file.

**Comment by davevad93** [comment, marked as answer]
Hi @Husam-AbuZina ,

It's actually possible to download the zip of an older repo version, to do so follow these steps:

1. Click on the `commit history` of the repo:

![0](https://github.com/community/community/assets/94308234/3573a7a3-cfed-49d2-8a81-897e7fea1390)

2. Choose an older commit and click on `View commit details`:

![1](https://github.com/community/community/assets/94308234/7810fec3-184d-4743-a023-b6e6cf9ee82d)

3. Click on `Browse files` in order to browse the repo at that point in the history:

![2](https://github.com/community/community/assets/94308234/23850849-c100-4960-a107-d6de5d1cc708)

4. As you can see, you are now browsing an older version of the repo:

![3](https://github.com/community/community/assets/94308234/6292a492-c6bf-4fac-aacb-fa63232fbb33)

5. Now you can download the zip of the older repo with all its contents by clicking on `Code` => `Download ZIP`

![4](https://github.com/community/community/assets/94308234/dbea6945-d1a0-4e93-ad89-3d9a1c68c408)

**Reply by Mearayrret** [reply]
I really wish that each and everyone people understands the risk involve in crypto business and try possible best to mitigate it. Every business has it's rules and regulations. Last year November, I foolishly invested £556,000 on a crypto platform, unknowingly to me that it was a scam. I tried all possible means to retrieve it back but ended up being scammed £78,788 for the second time. I became so frustrated and reported the transaction to appropriate authorities but no positive response. After viewing the testimony of one Mrs Jason on how she was able to retrieve her fund from the Fake crypto platform with the help of <REVOKETECH >>>,on Tele gram  I gave it a try by contacting techrevoke@gmail.com and was asked to forward all proof of transaction to the email. After 1days they were able to retrieve all the funds for me.

**Comment by Mearayrret** [comment]
I really wish that each and everyone people understands the risk involve in crypto business and try possible best to mitigate it. Every business has it's rules and regulations. Last year November, I foolishly invested £556,000 on a crypto platform, unknowingly to me that it was a scam. I tried all possible means to retrieve it back but ended up being scammed £78,788 for the second time. I became so frustrated and reported the transaction to appropriate authorities but no positive response. After viewing the testimony of one Mrs Jason on how she was able to retrieve her fund from the Fake crypto platform with the help of <REVOKETECH >>>,on Tele gram  I gave it a try by contacting techrevoke@gmail.com and was asked to forward all proof of transaction to the email. After 1days they were able to retrieve all the funds for me.

**Comment by Husam-AbuZina** [comment]
Thanks @davevad93, it Workds!

**Reply by davevad93** [reply]
You're welcome @Husam-AbuZina !
